### PR TITLE
Route legitimacy check

### PR DESCRIPTION
**Pull request by georgedigkas**

### CODE REVIEW

**Review by longbowlu — COMMENTED** [review]
we need to do the same thing in `transferTokensWithSignatures` right?

**Comment by georgedigkas** [comment]
> we need to do the same thing in `transferTokensWithSignatures` right?

You are right! Just added it

File: contracts/utils/MessageVerifier.sol [hunk 1]
```diff
@@ -31,4 +31,17 @@ abstract contract MessageVerifier is Initializable {
         }
         _;
     }
+
+    modifier verifyDestinationChainID(uint8 destinationChainID) {
+        // Check that destination chain ID is valid
+        require(
+            destinationChainID == BridgeMessage.SUI ||
```
Comment (Bridgerz) [inline contracts/utils/MessageVerifier.sol]: You'll want to check the destination matches what we have in [notion](https://www.notion.so/mystenlabs/Native-Bridge-V1-4ed9dd1877d042848f72148302062bf4?pvs=4#7acae2bb98cb4eb0854ed5fca7c1fe9f)

Comment (longbowlu) [reply contracts/utils/MessageVerifier.sol]: and the check is pair-wise. I added all eligible routes in that page

**Pull request closed without merging**